org.eclipse.xtext.ide.tests/testlang-src/org/eclipse/xtext/ide/tests/testlanguage/TestLanguageRuntimeModule.xtend

```diff
@@ -26,9 +28,13 @@ class TestLanguageRuntimeModule extends AbstractTestLanguageRuntimeModule {
     def Class<? extends ISignatureHelpService> bindSignatureHelpService() {
         return SignatureHelpServiceImpl;
     }
-    
+
     def Class<? extends IColoringService> bindIColoringService() {
     	return ColoringServiceImpl;
     }
 
+	def Class<? extends ISemanticHighlightingCalculator> bindISemanticHighlightingCalculator() {
```

Comment:
Mixed indentations

Reply:
It was already mixed, I'll switch to tabs.